### ISSUE TITLE
Improvement? Allow /profit /balance /performance /status+table when bot is stopped

### ISSUE THREAD

**Issue by seansan** [gcarq/freqtrade#326]
Improvement? Allow /profit /balance /performance /status+table when bot is stopped

We see that when the BOT is in status stopped (read: not actively trading) that we cant access any commands. E.g. we noticed this when we STOP the bot on sudden BTC swings ... but we still want to see our status for example. 

Suggestion: should we not change the stop and start commands to read like "start=active trading", "stop=stop trading", but the bot itself is always on? We can then make a /restart command to restart the code itself (what stop, start does in effect)

In short: 
- change stop and start to influence the trading itself
- always allow other "informational" commands
- (option) create new option /restart to actually reload the code base?
- start can also work as follows: suppose bot is down, then in that case try a code start first before a trading start

start command: check if bot is up, if not then start, start trading
stop command: stop trading
restart command: stops bot, starts bot code, then start trading

Idea?
